Maven project rultor-conveyer  got fail (in Windows)

Env : Jdk 7
Please check below exception,

```
       |   [ERROR]  conveyer-1 com.rexsl.test.BufferedJerseyFetcher: #fetch(): t
hrown java.lang.AssertionError(com.sun.jersey.api.client.ClientHandlerException:
 java.net.SocketTimeoutException: connect timed out) out of com.rexsl.test.Buffe
redJerseyFetcher#fetch_aroundBody4[125] in 5s
       |   [ERROR]  conveyer-1 com.rexsl.test.BufferedJerseyFetcher: #body(): th
rown java.lang.AssertionError(com.sun.jersey.api.client.ClientHandlerException:
java.net.SocketTimeoutException: connect timed out) out of com.rexsl.test.Buffer
edJerseyFetcher#fetch_aroundBody4[125] in 5s
       |   [ERROR]  conveyer-1 com.rexsl.test.JerseyTestResponse: #getBody(): th
rown java.lang.AssertionError(com.sun.jersey.api.client.ClientHandlerException:
java.net.SocketTimeoutException: connect timed out) out of com.rexsl.test.Buffer
edJerseyFetcher#fetch_aroundBody4[125] in 5s
       |   CONSOLE:   0:06 SEVERE #fetch(): thrown java.lang.AssertionError(com.
sun.jersey.api.client.ClientHandlerException: java.net.SocketTimeoutException: c
onnect timed out) out of com.rexsl.test.BufferedJerseyFetcher#fetch_aroundBody4[
125] in 5s
       |   CONSOLE:   0:06 SEVERE #body(): thrown java.lang.AssertionError(com.s
un.jersey.api.client.ClientHandlerException: java.net.SocketTimeoutException: co
nnect timed out) out of com.rexsl.test.BufferedJerseyFetcher#fetch_aroundBody4[1
25] in 5s
       |   CONSOLE:   0:06 SEVERE #getBody(): thrown java.lang.AssertionError(co
m.sun.jersey.api.client.ClientHandlerException: java.net.SocketTimeoutException:
 connect timed out) out of com.rexsl.test.BufferedJerseyFetcher#fetch_aroundBody
4[125] in 5s
       |   CONSOLE:   0:06 INFO ?em?ly 'XPATH "/snapshot[not(stdout)]"; STRICT "
1"; ADD "stdout"; SET "http://localhost:9095/d72cc44be585100d1a765222126fd583";'

       |   CONSOLE:   0:06 INFO ?em?ly 'XPATH "/snapshot[not(version)]"; STRICT
"1"; ADD "version"; ADD "name"; SET "1.0-SNAPSHOT"; UP; ADD "revision"; SET "";
UP; ADD "date"; SET "2013-09-10 17:44";'
       |   CONSOLE:   0:06 INFO nothing to do
       |   CONSOLE:   0:06 INFO ?em?ly 'XPATH "/snapshot/stdout"; STRICT "1"; RE
MOVE;'
       |   CONSOLE:   0:06 INFO ?em?ly 'XPATH "/snapshot[not(finish)]"; STRICT "
1"; ADD "finish"; SET "2013-09-10T12:33:07Z";'
       |   CONSOLE:   0:06 INFO ?em?ly 'XPATH "/snapshot[not(duration)]"; STRICT
 "1"; ADD "duration"; SET "5438";'
       |   CONSOLE:   0:06 INFO execution stopped after 5s and 5 tick(s)
       |   INFO: main waiting 0ms for threads termination
       |   [WARN]  front-1 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.net.SocketException: socket closed')
       |   INFO: main #shutdown(): succeeded
       |   WARN: main #shutdown(): failed
       |   ERROR: back-17 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-3 #dispatch(): thrown java.lang.InterruptedException out
of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   [WARN]  back-3 com.jcabi.log.VerboseRunnable: swallowed exception: ja
va.lang.IllegalStateException('java.lang.InterruptedException')
       |   ERROR: back-31 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-15 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-13 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   [WARN]  back-31 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-13 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-15 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   ERROR: back-29 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-20 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-21 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   [WARN]  back-29 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-20 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-21 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   ERROR: back-27 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-5 #dispatch(): thrown java.lang.InterruptedException out
of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-25 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-22 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   [WARN]  back-27 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-25 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-5 com.jcabi.log.VerboseRunnable: swallowed exception: ja
va.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-22 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   ERROR: back-12 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-26 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-10 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   [WARN]  back-12 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-26 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   ERROR: back-28 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-7 #dispatch(): thrown java.lang.InterruptedException out
of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   [WARN]  back-10 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-28 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   ERROR: back-30 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-6 #dispatch(): thrown java.lang.InterruptedException out
of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   [WARN]  back-7 com.jcabi.log.VerboseRunnable: swallowed exception: ja
va.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-30 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-6 com.jcabi.log.VerboseRunnable: swallowed exception: ja
va.lang.IllegalStateException('java.lang.InterruptedException')
       |   ERROR: back-4 #dispatch(): thrown java.lang.InterruptedException out
of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   [WARN]  back-4 com.jcabi.log.VerboseRunnable: swallowed exception: ja
va.lang.IllegalStateException('java.lang.InterruptedException')
       |   ERROR: back-1 #dispatch(): thrown java.lang.InterruptedException out
of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-2 #dispatch(): thrown java.lang.InterruptedException out
of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-24 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   [WARN]  back-17 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   ERROR: back-14 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   [WARN]  back-24 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   ERROR: back-19 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   [WARN]  back-1 com.jcabi.log.VerboseRunnable: swallowed exception: ja
va.lang.IllegalStateException('java.lang.InterruptedException')
       |   ERROR: back-9 #dispatch(): thrown java.lang.InterruptedException out
of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   [WARN]  back-14 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-9 com.jcabi.log.VerboseRunnable: swallowed exception: ja
va.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-19 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   ERROR: back-23 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-18 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-11 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-16 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-8 #dispatch(): thrown java.lang.InterruptedException out
of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   ERROR: back-32 #dispatch(): thrown java.lang.InterruptedException out
 of java.util.concurrent.SynchronousQueue#take[929] in 1min
       |   [WARN]  back-2 com.jcabi.log.VerboseRunnable: swallowed exception: ja
va.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-18 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-16 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-11 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-23 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-32 com.jcabi.log.VerboseRunnable: swallowed exception: j
ava.lang.IllegalStateException('java.lang.InterruptedException')
       |   [WARN]  back-8 com.jcabi.log.VerboseRunnable: swallowed exception: ja
va.lang.IllegalStateException('java.lang.InterruptedException')
       |   INFO: main #shutdown(): shutdownNow() succeeded
       |   INFO: main #close(): in 1s
       |   INFO: main #close(): in 1min
       |   [INFO] --------------------------------------------------------------
----------
       |   [INFO] BUILD SUCCESS
       |   [INFO] --------------------------------------------------------------
----------
       |   [INFO] Total time: 1:25.578s
       |   [INFO] Finished at: Tue Sep 10 18:04:23 IST 2013
       |   [INFO] Final Memory: 6M/15M
       |   [INFO] --------------------------------------------------------------
----------
       |   Running post-build script: D:\Shailendra\pp\projects\tcp2\rultor\rult
or-conveyer\target\it\basic\verify.groovy
       D:\Shailendra\pp\projects\tcp2\rultor\rultor-conveyer\target\it\basic\bui
ld.log
[INFO] -------------------------------------------------
[INFO] Build Summary:
[INFO]   Passed: 0, Failed: 1, Errors: 0, Skipped: 0
[INFO] -------------------------------------------------
[ERROR] The following builds failed:
[ERROR] *  basic\pom.xml
[INFO] -------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] rultor-conveyer ................................... FAILURE [2:02.641s]
[INFO] rultor-client ..................................... SKIPPED
[INFO] rultor-web ........................................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2:03.469s
[INFO] Finished at: Tue Sep 10 18:04:24 IST 2013
[INFO] Final Memory: 25M/61M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-invoker-plugin:1.8
:run (default) on project rultor-conveyer: 1 build failed. See console output ab
ove for details. -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e swit
ch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please rea
d the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureExc
eption
```

--- a/rultor-conveyer/src/it/basic/verify.groovy
+++ b/rultor-conveyer/src/it/basic/verify.groovy
@@ -30,9 +30,10 @@
 
 def log = new File(basedir, 'build.log')
 assert log.exists()
-assert log.text.contains('INFO: main #start():')
-assert log.text.contains('CONSOLE: ')
-assert log.text.contains('INFO χemβly ')
-assert log.text.contains('INFO nothing to do')
-assert log.text.contains('INFO: main #close():')
+def text = log.getText('UTF-8')
+assert text.contains('INFO: main #start():')
+assert text.contains('CONSOLE: ')
+assert text.contains('INFO χemβly ')
+assert text.contains('INFO nothing to do')
+assert text.contains('INFO: main #close():')
 

--- a/rultor-conveyer/src/main/java/com/rultor/conveyer/fake/Console.java
+++ b/rultor-conveyer/src/main/java/com/rultor/conveyer/fake/Console.java
@@ -34,8 +34,10 @@ import com.rultor.spi.Pageable;
 import com.rultor.tools.Time;
 import java.io.IOException;
 import java.io.InputStream;
+import java.io.PrintStream;
 import lombok.EqualsAndHashCode;
 import lombok.ToString;
+import org.apache.commons.lang.CharEncoding;
 
 /**
  * Drain to console.
@@ -64,8 +66,11 @@ public final class Console implements Drain {
     @Override
     @SuppressWarnings("PMD.SystemPrintln")
     public void append(final Iterable<String> lines) throws IOException {
+        final PrintStream stream = new PrintStream(
+            System.out, true, CharEncoding.UTF_8
+        );
         for (String line : lines) {
-            System.out.println(String.format("CONSOLE: %s", line));
+            stream.println(String.format("CONSOLE: %s", line));
         }
     }
 

